kinder/ci/tools/update-workflows/templates/workflows/external-ca-tasks.yaml

```diff
@@ -114,6 +114,7 @@ tasks:
       - --name={{ .vars.clusterName }}
       - --loglevel=debug
       - --kubeadm-verbosity={{ .vars.kubeadmVerbosity }}
+      - --ignore-preflight-errors=SystemVerification
```

Comment:
shall we use variable here?  just like https://github.com/kubernetes/kubeadm/blob/cfc3ba886ce28a70ca263cda15a9b46286b8df4a/kinder/ci/workflows/skew-x-on-y-tasks.yaml#L19

Reply:
Maybe that's okay, because it only appear once